rakefile.rb

```diff
@@ -37,11 +37,6 @@
   if args.to_hash.values.include? "p"
     update_pksim(versionId)
   end
-  if args.to_hash.values.include? "r"
-    update_ospsuite_r(versionId)
-  end
-  
-

```

Comment:
I'm getting rid of the scripts that were used to update OSPSuite-R with fresh binaries. We might need this local update stuff, but it would have to be implemented in PK-Sim rakefile.